src/content/other/electric-cars.mdx

```diff
@@ -1,15 +1,9 @@
 ---
 title: Electric Cars
 description: Transportation is one of the largest sources of carbon emissions in the US
-sectionHeader: Cars, Trucks, and Things That Go
+sectionHeader: Electric Cars
 ---
 
-With transportation being one of the largest sources of carbon emissions in the US, the appeal of electric cars is very real.
-Electric cars are much more efficient than traditional vehicles when it comes to carbon emissions per mile as they run on electricity rather than gasoline.
-Right now is a a great time to invest in an EV if in the US, as there are tons of legislation currently getting passed to promote them.
-Additionally, all major electric car brands now support the Tesla supercharger network, so charging is less a worry than ever!
-Consumers who purchase EVs are eligible for a $7,500 tax credit on new EVs or a $4,000 tax credit for used EVs.
-States like California have made plans to ban the sale of all new gas powered vehicles by 2035.
-Overall, it’s a great time to invest in a electric car.
+With transportation being one of the largest sources of carbon emissions in the US, the appeal of electric cars is very real. Electric cars are much more efficient than traditional vehicles when it comes to carbon emissions per mile as they run on electricity rather than gasoline. Right now is a great time to invest in an EV in the US, as there are tons of legislation currently getting passed to promote them. Additionally, all major electric car brands now support the Tesla supercharger network, so charging is less of a worry than ever! Consumers who purchase EVs are eligible for a $7,500 tax credit on new EVs or a $4,000 tax credit for used EVs. States like California have made plans to ban the sale of all new gas-powered vehicles by 2035. Overall, it’s a great time to invest in an electric car.
```

Comment:
A few wording changes, but mostly just that it should be wrapped.
I think the only major thing was the use of a ton, which you inherited from me (I think). It feels too casual. Does this version sound better?

```suggestion
With transportation being one of the largest sources of carbon emissions in the US, the appeal of electric cars is very real.
Electric cars are much more efficient than traditional vehicles when it comes to carbon emissions per mile, as they run on electricity rather than gasoline.
Right now is a great time to invest in an EV in the US, as there is a lot of legislation currently in the process of getting passed to promote them.
Additionally, all major electric car brands now support the Tesla supercharger network, so charging is less of a worry than ever!
Consumers who purchase EVs are eligible for a $7,500 tax credit on new EVs or a $4,000 tax credit for used EVs.
States like California have made plans to ban the sale of all new gas-powered vehicles by 2035.
Overall, it’s a great time to invest in an electric car.
```